lib/metrics.js

```diff
@@ -15,67 +28,43 @@ module.exports = {
 function prepareData(res) {
   const audits = res.audits;
 
-  const resFMP = audits['first-meaningful-paint'];
-  const resFMPext = resFMP.extendedInfo;
-  const resSI = audits['speed-index-metric'];
-  const resSIext = resSI.extendedInfo;
-  const resTTI = audits['time-to-interactive'];
-  const resTTIext = resTTI.extendedInfo;
-
   const colorP0 = 'yellow';
   const colorP2 = 'green';
   const colorVisual = 'blue';
 
-  const timestamps = [{
-    title: 'Navigation Start',
-    name: 'navstart',
-    value: resFMPext && resFMPext.value.timestamps.navStart
-  }];
+  const timings = [];
+  metricsDefinitions.forEach(metric => {
+    try {
+      const resolvedMetric = {
+        title: metric.name,
+        name: metric.id,
+        value: metric.getTs(audits)
```

Comment:
There _could_ be `getMetricTiming` method.. but it'd have to take the navStart as well as the metric. something like:
```js
static getMetricTiming(navStart, metric) {
  return (metric.getTs() - navStart.getTs()) / 1000;
}
```

I'm fine with adding that as a utility method to pwmetrics-events. But also fine with us handling that over here.


Over here we'd tweak things a tad:

```js
const timings = [];
const navStart = metricsDefinitions.find(def => def.id === metricsIds.NAVSTART); 

metricsDefinitions.forEach(metric => {
    try {
      const ts = metric.getTs(audits);
      const resolvedMetric = {
        name: metric.name,
        id: metric.id,
        timestamp: metric.getTs(audits),
        timing: (metric.getTs(audits) - navStart.getTs(audits)) / 1000;
      };
// ...
```

I favor this second approach a bit. How does that look to you?